packages/react-components/src/molecules/__tests__/ErrorCard.test.tsx

```diff
@@ -6,7 +6,7 @@ import ErrorCard from '../ErrorCard';
 jest.useFakeTimers('modern');
 
 it('renders a plain message', () => {
-  const { getByText } = render(<ErrorCard>oops</ErrorCard>);
+  const { getByText } = render(<ErrorCard title="oops" description="" />);
```

Comment:
This can't be all the test changes required surely? You added new props and stuff